modules/_canvas-kit-react/package.json

```diff
@@ -36,34 +36,34 @@
     "react": ">= 16.8 < 17"
   },
   "dependencies": {
-    "@workday/canvas-kit-react-action-bar": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-avatar": "^3.0.0-alpha.8",
-    "@workday/canvas-kit-react-banner": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-button": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-card": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-checkbox": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-color-picker": "^3.0.0-alpha.9",
-    "@workday/canvas-kit-react-common": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-cookie-banner": "^3.0.0-alpha.7",
-    "@workday/canvas-kit-react-core": "^3.0.0-alpha.7",
+    "@workday/canvas-kit-react-action-bar": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-avatar": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-banner": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-button": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-card": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-checkbox": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-color-picker": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-common": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-cookie-banner": "^3.0.0-beta.0",
+    "@workday/canvas-kit-react-core": "^3.0.0-beta.0",
     "@workday/canvas-kit-react-fonts": "^3.0.0-alpha.6",
```

Comment:
I wonder if we should do something to prompt a bump for this guy...

Reply:
Personally, I think all packages should bump regardless of changes. I think this will make it easier to consume. Less questions like "I bumped all the canvas-kit package to the latest beta, but yarn/npm is failing telling me the `@workday/canvas-kit-x` package doesn't exist"